### PR TITLE
Add process readiness changed controller and event

### PR DESCRIPTION
**Pull request by ameowlia**
See more context here: https://github.com/cloudfoundry/capi-release/issues/363

* [x] I have reviewed the [contributing guide](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/CONTRIBUTING.md)

* [x] I have viewed, signed, and submitted the Contributor License Agreement

* [x] I have made this pull request to the `main` branch

* [x] I have run all the unit tests using `bundle exec rake`

* [x] I have run [CF Acceptance Tests](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/spec/README.md#cf-acceptance-tests-cats)

### CODE REVIEW

**Comment by sethboyles** [comment]
is this waiting on any other PRs before we can merge?

**Comment by ameowlia** [comment]
@matt-royal Nope! Just waiting for a review.

**Comment by ameowlia** [comment]
@sethboyles - Nope! Just waiting for a review!

File: app/repositories/app_event_repository.rb
Comment (dalvarado) [inline app/repositories/app_event_repository.rb:18]: hey @ameowlia, where is this function being called, i couldn't find it?

Comment (ameowlia) [reply app/repositories/app_event_repository.rb:18]: My guess is that I forgot to delete that code. I initially (erroneously) added an app event instead of an app.process event. It's probably part of the app event that I should've deleted. I'll test to make sure everything still works if I remove it tomorrow. Good catch!

Comment (ameowlia) [reply app/repositories/app_event_repository.rb]: I removed the code. Units still pass. And when I deployed it everything still works.

**Review by dalvarado — APPROVED** [review]

**Review by moleske — APPROVED** [review]
woot!

**Pull request merged**
